shared/productTypes.ts

```diff
@@ -579,6 +581,54 @@ export const PRODUCT_TYPES: Record<ProductTypeKeys, ProductType> = {
 			swapFeedbackAndContactUs: true,
 		},
 	},
+	voucherobserver: {
+		productTitle: () => 'Newspaper Voucher - Observer',
```

Comment:
if this is a key value that relates to MDAPI responses, we should have it once in a const, and then reference it in all the places it's used.  Or maybe it's a mixture of MDAPI and display names is it?  e.g. I just wouldn't know which to change if it changed in MDAPI for whatever reason.